### DISCUSSION TITLE
display more details in accesslog

### DISCUSSION THREAD

**Discussion by tcodehuber** [juicedata/juicefs#2551] [Ideas]
now there are only uid and gid related contents, such as belows:
![image](https://user-images.githubusercontent.com/5411862/184078886-b1a20ff2-1d72-4a5a-94c0-ca5f16083509.png)

But for hadoop admins, it is very important and convenient to see username and groupname in accesslog files. Look forward to coming true in the future.

**Comment by davies** [comment]
JuiceFS use uid and gid for permission checking, it will be expensive (may also not accurate) to translate the uid/gid to username for logging.
